src/pages/AccountDetailsPage.vue

```diff
@@ -1,20 +1,20 @@
 <script setup lang="ts">
 import { useRouter } from 'vue-router';
 import { computed, onMounted, ref } from 'vue';
-import { http } from '@/shared/api';
+import * as http from '@/shared/api';
 import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
 import DataField from '@/shared/ui/components/DataField.vue';
 import { useTable } from '@/shared/lib/table';
 import BaseTable from '@/shared/ui/components/BaseTable.vue';
 import { type filterTransactionsModel as ftm, TransactionStatusFilter } from '@/features/filter-transactions';
 import BaseHash from '@/shared/ui/components/BaseHash.vue';
 import TransactionStatus from '@/entities/transaction/TransactionStatus.vue';
-import { transactionModel } from '@/entities/transaction';
 import { useWindowSize } from '@vueuse/core';
 import { format } from '@/shared/lib/time';
 import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
 import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
-import invariant from 'tiny-invariant';
+import type { Account } from '@/shared/api/schemas';
+import { AccountIdSchema } from '@/shared/api/schemas';
```

Comment:
```suggestion
import { type Account, AccountIdSchema } from '@/shared/api/schemas';
```

Reply:
This is ESLint approach to separate imported types and values. Should I really manually change it or maybe there is new eslint rule?

Reply:
If ESLint does it automatically, than it's fine.

Maybe if we change rules it can patch it automatically though.